x-pack/plugin/sql/qa/build.gradle

```diff
@@ -96,20 +94,20 @@ subprojects {
 
   if (project.name != 'security') {
     // The security project just configures its subprojects
+    apply plugin: 'elasticsearch.testclusters'
     apply plugin: 'elasticsearch.rest-test'
 
-    integTestCluster {
+    testClusters.integTest {
+      distribution = 'DEFAULT'
       setting 'xpack.monitoring.enabled', 'false'
       setting 'xpack.ml.enabled', 'false'
       setting 'xpack.watcher.enabled', 'false'
-      setting 'script.max_compilations_rate', '1000/1m'
     }
 
-    task runqa(type: RunTask) {
-      setting 'xpack.monitoring.enabled', 'false'
-      setting 'xpack.ml.enabled', 'false'
-      setting 'xpack.watcher.enabled', 'false'
-      setting 'script.max_compilations_rate', '1000/1m'
+    task runqa {
+      doFirst {
+        println "Run with `-Dtestclusters.inspect.failure=true integTest` to leave the cluster running after failure"
```

Comment:
Is this just instructing folks to use the `run` task instead? If so should we be more specific? Also, if we are effectively "deprecating" this task, we should do so in a `taskGraphReady` callback so that we catch this scenario earlier for the user.

Reply:
These projects had something similar to a run that stared up  cluster identical to the testcluster. We have no equivalent of `run` yet and that still uses the old method. 
I suspect the intent was to ease debugging , and that's exactly what the command `-Dtestclusters.inspect.failure=true integTest` does, it runs tests and leaves the cluster running if it fails. I don't think it's worth looking at the task graph as I expect the task to be run directly for a single project, so doing it via the task graph would not yield a significant change  with that use-case.